src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -1304,10 +1098,15 @@ internal static class DbConnectionStringKeywords
         internal const string MaxPoolSize = "Max Pool Size";
         internal const string Pooling = "Pooling";
         internal const string MinPoolSize = "Min Pool Size";
+        internal const string PoolBlockingPeriod = "Pool Blocking Period";
     }
 
     internal static class DbConnectionStringSynonyms
     {
+#if NETFRAMEWORK
+        //internal const string TransparentNetworkIPResolution = TRANSPARENTNETWORKIPRESOLUTION;
+        internal const string TRANSPARENTNETWORKIPRESOLUTION = "transparentnetworkipresolution";
+#endif
```

Comment:
Weird casing behaviour. If these are strictly internal strings we could audit them to see why they're forced upper and lower case and see if we can normalize the behaviour somehow.

Reply:
We've ignored the casing in string comparison for connection properties. It was camel-case in some cases using `NetCore` while `NetFx` used lower-case. I tried to keep them unified by porting from 'NetFx' and having the lowest impact on code changes for the sake of review.
My personal preference would be using camel-case cause of more legibility if we managed to modify them.

Reply:
>  having the lowest impact on code changes for the sake of review.

When I've tried this it hasn't helped. Unless you're absolutely sure you'll have chance to do a second pass over the files involved it's better to make everything as coherent as possible in the ones you touch. It's better to change small numbers of files to their best version than work on more files incompletely.

Reply:
We already have related functional tests which were passed. We could rich those tests If you think we've missed some tests here.